resources/lib/ui/embed_extractor.py

```diff
@@ -97,7 +97,7 @@ def playSource():
         reqObj = http.send_request(playUrl)
         return VIDEO_RE.findall(reqObj.text)[0]
 
-    return playSource
+    return playSource()
```

Comment:
this was intended i dont want it to actually resolve before user selected.
so we are getting all possible options using 1 request.
then only when user selects a specific source it will send 1 more request to resolve it finally.

so this thing should be resolved on config and not here.

Reply:
after seeing the other PR (#209 )
i realized this playSource thingy is for there.
please just remove it here and we will merge this PR :)
and let's tackle it there.